ext/ruby_binlog.cpp

```diff
@@ -411,6 +423,7 @@ struct Client {
     rb_define_method(rb_cBinlogClient, "position=", __F(&set_position2), 1);
     rb_define_method(rb_cBinlogClient, "get_position", __F(&get_position), -1);
     rb_define_method(rb_cBinlogClient, "position", __F(&get_position2), 0);
+    rb_define_method(rb_cBinlogClient, "set_ssl_ca", __F(&set_ssl_ca), 1);
```

Comment:
This is fine.  I just wonder what if you pass ssl_ca as an optional hash argument of `initialize`.  A little cleaner, maybe?  Future expansion proof for sure.
```
client = Binlog::Client.new(mysql_url, ssl_ca: ssl_ca)
```